app/components/alert_component.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class AlertComponent < ApplicationComponent
+  def initialize(
+    scheme: :default,
+    title: nil,
+    icon: nil,
+    **kwargs
+  )
+    @scheme = scheme
+    @title = title
+    @icon = icon
+
+    super(data: data, **kwargs)
+  end
+
+  private
+
+  def bg_class
```

Comment:
Maybe these would be better as hashes, instead of methods? 🤷🏼‍♀️

Reply:
No idea, let's see how it goes for a bit.